packages/wallet/src/redux/indirect-funding/reducer.ts

```diff
@@ -32,7 +32,14 @@ function playerBReducer(
   state: states.Initialized,
   action: actions.indirectFunding.Action,
 ): states.Initialized {
-  return state;
+  return {
+    ...state,
+    indirectFunding: states.indirectFunding.playerB.waitForDirectFunding({
+      ledgerId: 'ledgerId',
```

Comment:
Maybe leave a TODO that this hardcoding get's removed? Or maybe a comment with a Zube card reference?

Reply:
I won't bother, because the `playerBReducer` is simply a placeholder to disappear when we write the actual playerB reducer

Reply:
👍